sales: iterate on closing a deal (#1231)
FlowFuse/website: src/handbook/sales/closing-a-deal.md

## Steps

1. Once the order form is signed by the customer, you should move the deal to Closed Won in HubSpot. You do not need to wait for countersignature.
1. Once all parties have signed, download the completed Order Form and Subscription Agreement, and upload them to the Legal folder in Google Drive.

> 1. If the license is self hosted, generate the key, then include in the onboarding email.

Author: TODO: Link to generating a license section

Author: ```suggestion
1. For self-managed customers a [license key needs to be generated](/handbook/sales/pricing/#generating-a-license) to be included in the onboarding email.
```

1. If it is a cloud deal, any discounts need to be setup by the CEO in Stripe. Let the CEO know which email to send the invoice to once configured.

> 1. For Cloud teams: Upgrade their team to the right tier 

Author: @robmarcer Didn't we also need to set a PrePaid thing in Stripe?
